### PR TITLE
Clarify `error` status

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: building/tooling/test-runners/interface.md [hunk 1]
```diff
@@ -46,7 +46,11 @@ The following overall statuses are valid:
 
 - `pass`: All tests passed
 - `fail`: At least one test has the status `fail` or `error`
-- `error`: No test was executed correctly (this usually means e.g. a compile error, or a syntax error)
+- `error`: No test was executed (this usually means e.g. a compile error, or a syntax error)
```
Comment (iHiD) [inline building/tooling/test-runners/interface.md]:
```suggestion
- `error`: No test was executed (this usually means a compile error or a syntax error)
```

(It's not very usual to use `e.g.` in the middle of a sentence)

File: building/tooling/test-runners/interface.md [hunk 2]
```diff
@@ -46,7 +46,11 @@ The following overall statuses are valid:
...
+
+To clarify, `error` should _only_ be used if **none of the tests were run**.
```
Comment (iHiD) [inline building/tooling/test-runners/interface.md]:
```suggestion
The `error` status should _only_ be used if **none of the tests were run**.
```

File: building/tooling/test-runners/interface.md [hunk 3]
```diff
@@ -46,7 +46,11 @@ The following overall statuses are valid:
...
+Usually, this only happens for compiled languages, as not being able to compile the code will prevent the tests from being run.
```
Comment (iHiD) [inline building/tooling/test-runners/interface.md]:
```suggestion
For compiled languages this is generally a result of the as code not being able to compile.
```

File: building/tooling/test-runners/interface.md [hunk 4]
```diff
@@ -46,7 +46,11 @@ The following overall statuses are valid:
...
+In most interpreted languages, the tests will run even with syntax errors in the source, as they'll only find that out at run time.
```
Comment (iHiD) [inline building/tooling/test-runners/interface.md]:
```suggestion
For interpreted languages, this is generally the result of a syntax error that stops the file from parsing.
```

File: building/tooling/test-runners/interface.md (hunk 3 above)
Comment (ErikSchierboom) [reply building/tooling/test-runners/interface.md]:
```suggestion
For compiled languages this is generally a result of the code not being able to compile.
```

**Pull request merged**